tests/e2e/dualstack/dualstack_test.go

```diff
@@ -211,3 +324,59 @@ func ImageFromRegistry(regexp string) types.GomegaMatcher {
 func getEnvVars(container corev1.Container) []corev1.EnvVar {
 	return container.Env
 }
+
+func getPodURL(version supportedversion.VersionInfo, namespace string) string {
```

Comment:
Maybe change the func name to getYAMLPodURL? Calling this func getPodURL may be confusing